src/lib/serviceRoutines/postQueryContext.cpp

```diff
@@ -282,13 +287,18 @@ static bool queryForward
                       &statusCode,
                       noHeaders,
                       mimeType);
-
+                
```

Comment:
```suggestion

```

(Avoid whitespace in blank line))

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3749/commits/4c99163abe5ff68a1ee5eb612d0e9a8c74de5958